dspace-api/src/main/java/org/dspace/disseminate/CoverPageService.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.disseminate;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.dspace.content.Item;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+
+/**
+ * This service renders a PDF coverpage for an item.
+ */
+public class CoverPageService {
+
+    private static final Logger LOG = LogManager.getLogger(CoverPageService.class);
+
+    private final PdfGenerator pdfGenerator = new PdfGenerator();
+
+    CoverPageService(
+            @Value("${citation-page.cover-template:dspace_coverpage}")
+            String coverTemplate,
+
+            @Autowired(required = false)
+            CoverPageContributor coverPageContributor
+    ) {
+        this.coverTemplate = coverTemplate;
+        this.coverPageContributor = coverPageContributor == null ?
+                new DefaultCoverPageContributor() :
+                coverPageContributor;
+    }
+
+    private final String coverTemplate;
+
+    private final CoverPageContributor coverPageContributor;
+
+    /**
+     * Render a PDF coverpage for the given Item. The implementation may use the context and
+     * any relevant meta data from the Item to populuate dynamic content in the rendered page.
```

Comment:
Typo here (populate). In addition, I suggest that a clarification that all metadata fields (using format schema_field_qualifier) are passed to coverPageContributor and can be referenced in HTML template (therefore eliminating the need to use an earlier configuration item citation-page.fields) would be added here (and summarized in example HTML template as well).